augur/tasks/init/celery_app.py

```diff
@@ -32,7 +32,8 @@
                 'augur.tasks.github.repo_info.tasks',
                 'augur.tasks.github.detect_move.tasks',
                 'augur.tasks.github.pull_requests.files_model.tasks',
-                'augur.tasks.github.pull_requests.commits_model.tasks']
```

Comment:
@IsaacMilarky / @ABrain7710 : This looks right. Just checking.

Reply:
Yep this is correct.